airflow/executors/celery_executor.py

```diff
@@ -182,10 +183,17 @@ def heartbeat(self):
         else:
             open_slots = self.parallelism - len(self.running)
 
-        self.log.debug("%s running task instances", len(self.running))
-        self.log.debug("%s in queue", len(self.queued_tasks))
+        num_running_tasks = len(self.running)
```

Comment:
Looks like we have similar code in:
https://github.com/apache/airflow/blob/a9eff7a7e4367414040c6fb1cf17f30bcaf0dd5c/airflow/executors/base_executor.py#L116-L130

Can we consolidate the two?

Reply:
@milton0825 Does current change look better to you?